CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 1.8.1-SNAPSHOT (YYYY-MM-DD)
```

Comment:
This should be 1.9.0

Reply:
It is ok, this PR should target releases.